betterproto/plugin.py

```diff
@@ -314,6 +314,24 @@ def generate_code(request, response):
                                     output["typing_imports"].add("Optional")
                             break
 
+                    mutable_default_args = []
+                    if (
+                        not method.client_streaming
+                        and input_message
+                        and input_message.get("properties")
+                    ):
+                        properties = []
+                        for f in input_message.get("properties"):
+                            if f["zero"] != "None" and (
+                                f["type"].startswith("List[")
+                                or f["type"].startswith("Dict[")
+                            ):
+                                output["typing_imports"].add("Optional")
+                                mutable_default_args.append((f["py_name"], f["zero"]))
+                                f["zero"] = "None"
+                            properties.append(f)
+                        input_message["properties"] = properties
```

Comment:
I'm a bit nervous to see the entire `properties` field being overridden, as it makes it not immediately clear if it is changed fundamentally or not. Since the values are dictionaries (objects), it should be possible to just modify them, instead of recreating the list?

Reply:
Good call. Fixed it.